src/d3/hapticplot.d3.ts

```diff
@@ -14,17 +14,21 @@ export class Hapticplot{
   init(container: HTMLElement | null, data: number[]){
     this.data = data;
     this.container = container;
-    // create a scale so that there is correspondence between data set and screen render
+    // create a scale so that there is correspondence between data set and screen render,
+    // a linear mapping of data set values to values from 0 to 10
     this.hscale = d3.scaleLinear();
     this.hscale.domain([0, d3.max(this.data) as number])  // max of dataset
       .range([0, 100]);
-    this.setupPoints('blue', 0.1);                        // linear mapping of data set values to values from 0 to 10
+    this.setupPoints('green', 0.05);
```

Comment:
This is probably stylistic preference but it might make sense to make this value a file level constant like
`const POINT_SIZE = 0.05`; tends to be more maintainable.

Reply:
I added `POINT_SIZE` `DEFAULT_COLOR` and `HOVER_COLOR` as file level constants in the init function, and updated their use throughout.